lib/test.dart

```diff
@@ -8,6 +8,7 @@ import 'package:path/path.dart' as p;
 
 import 'src/backend/declarer.dart';
 import 'src/backend/invoker.dart';
+import 'src/backend/platform_selector.dart' show PlatformSelector; // for the dartdocs
```

Comment:
@lrhn Now that dartdoc autoimports everything under the sun, that may not be worth it anymore. For Flutter, at least, we wouldn't want to have imports that are only used by dartdocs.